src/build.rs

```diff
@@ -315,6 +338,7 @@ fn create_disk_image(
     bin_name: &str,
     config: &Config,
     mut kernel: File,
+    mut maybe_package: Option<File>,
```

Comment:
You don't pass the additional parameter currently, that's why the travis build fails. I think you accidentally overwrote your changes when merging the master branch.